lib/cloudkeeper/aws/core_connector.rb

```diff
@@ -0,0 +1,90 @@
+module Cloudkeeper
+  module Aws
+    # Class implementing GRPC procedures
+    class CoreConnector < CloudkeeperGrpc::Communicator::Service
```

Comment:
Please rework appliance update into two separate methods according to new proto file (sorry about that).

Reply:
No need to be sorry, I'll look into that